pkg/schemapatcher/gen_integration_test.go

```diff
@@ -67,7 +67,7 @@ var _ = Describe("CRD Patching From Parsing to Editing", func() {
 		var crdSchemaGen genall.Generator = &Generator{
 			ManifestsPath: "./valid",
 		}
-		rt, err := genall.Generators{&crdSchemaGen}.ForRoots("./...")
+		rt, err := genall.Generators{&crdSchemaGen}.ForRoots("./apis/kubebuilder/...", "./apis/legacy/...")
```

Comment:
same. I think it needs to be reverted to ensure that the changes made here do not break any current test. Could you please clarify why this change was required?